modules/apps/frontend-js/frontend-js-tabs-support-web/src/main/resources/META-INF/resources/index.d.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+export class TabsProvider {
+	constructor();
+
+	hide(menu: HTMLElement, trigger: HTMLElement): void;
+
+	show(menu: HTMLElement, trigger: HTMLElement): void;
+}
```

Comment:
Since nothing is returned from our exported function, I believe this would just be

```ts
export default function main(): void;
```